perf/docker/kafka/client.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
Please addd license to the top of this file similar to the one in our helloworld app:
https://github.com/istio/istio/blob/master/samples/helloworld/src/app.py#L1